contracts/KeysManager.sol

```diff
@@ -66,38 +67,42 @@ contract KeysManager is IKeysManager {
             isVotingActive: false,
             isPayoutActive: false
         });
+        successfulValidatorClone[_masterOfCeremony] = true;
+        Migrated("miningKey", _masterOfCeremony);
         if (_previousKeysManager != address(0)) {
             previousKeysManager = _previousKeysManager;
             KeysManager previous = KeysManager(previousKeysManager);
             initialKeysCount = previous.initialKeysCount();
         }
     }
 
-    function migrateFromPreviousKeysManager(address _miningKey, address _initialKey) public {
+    function migrateMiningKey(address _miningKey) public {
```

Comment:
We don't migrate miningKey history here. So, we'll lose history of ballots created with previous mining keys. I'm not sure, this is important. Just not to forget.